Shared/Services/ConfigLoader.swift

```diff
@@ -202,13 +203,27 @@ public struct ConfigLoader {
             throw ConfigLoaderError.CannotSaveOnboardingIsCompleted
         }
     }
-    
+
+    public func hideBackupBanner() throws -> Void {
```

Comment:
This does not hide the backups banner actually, maybe a better name will improve this, something like `shouldDisplayBackupsBanner(shouldDisplay: Bool)`?